client/state/jetpack-sync/selectors.js

```diff
@@ -81,11 +81,14 @@ function isFullSyncing( state, siteId ) {
  * @return {Number}          The percentage of sync completed, expressed as an integer
  */
 function getSyncProgressPercentage( state, siteId ) {
-	const syncStatus = getSyncStatus( state, siteId );
-	const queued = get( syncStatus, 'queue' );
-	const sent = get( syncStatus, 'sent' );
-
-	if ( isPendingSyncStart( state, siteId ) || ! queued || ! sent ) {
+	const syncStatus = getSyncStatus( state, siteId ),
+		queued = get( syncStatus, 'queue' ),
```

Comment:
The main advantages ar

- I don't have to handle the edge case of `syncStatus` being `undefined`, and then getting an error for trying to access a property of an undefined object
- I use `get` throughout Jetpack sync, so consistency's sake?